generateBuildrootSBOM.py

```diff
@@ -35,38 +33,36 @@
 # Buildroot manifest.csv file header shows the following header row
 # PACKAGE,VERSION,LICENSE,LICENSE FILES,SOURCE ARCHIVE,SOURCE SITE,DEPENDENCIES WITH LICENSES
 #
+from cyclonedx.model.component import ComponentType
+
 
-def create_buildroot_sbom(args, parser: br_parser):
+def create_buildroot_sbom(args, br_bom):
+    br_bom_local: cyclonedx.model.bom = br_bom
     #
     # Capture the components that describe the complete inventory of first-party software
-    final_component_details = list("")
     # Buildroot CSV file supplies software package data in each row. Any change to that map of data will break
-    # the resulting JSON. Thus a try/except block to help with run time issues.
+    # the resulting JSON. Use a try/except block to help with run time issues.
     with open(args.input_file, newline='') as csvfile:
         sheetX = csv.DictReader(csvfile)
+        my_component_list = []
```

Comment:
what is the purpose of `my_component_list`? 
it is never read.

Reply:
That was a left over object that is not needed. Removed. Thanks for catching that.